test/adapter/json/test_json_serialization.py

```diff
@@ -18,7 +18,18 @@
     example_aas_mandatory_attributes, example_submodel_template, create_example
 
 
-JSON_SCHEMA_FILE = os.path.join(os.path.dirname(__file__), 'aasJSONSchema.json')
+JSON_SCHEMA_FILE = os.path.join(os.path.dirname(__file__), '../schemas/aasJSONSchema.json')
+
+
+def skip_if_schema_not_exists():
+    def decorator(test_method):
+        def wrapper(self, *args, **kwargs):
+            if not os.path.exists(JSON_SCHEMA_FILE):
+                self.skipTest(f"JSON Schema does not exist at {JSON_SCHEMA_FILE}, skipping test")
+            else:
+                return test_method(self, *args, **kwargs)
+        return wrapper
+    return decorator
```

Comment:
Instead of decorating each individual schema test function we might as well skip the entire `JsonSerializationSchemaTest` class, i.e. something similar to `setUpClass` here: https://stackoverflow.com/a/37888952/4780052
Same for the `XMLSerializationSchemaTest`.

BTW Something else I only recently learned about is `functools.wraps`. When decorating a function, that function is replaced by the function returned by the decorator. This causes the function to have a different name, docstring and annotation at runtime. This is why `functools.wraps` should be used if possible. It copies the name, docstring and other attributes to the function returned by the decorator. See also: https://stackoverflow.com/questions/308999/what-does-functools-wraps-do

Not that it's relevant here, as the `setUpClass` way is the better approach here anyway IMO.

Reply:
You're right, this is the cleaner way. I'll adapt the tests accordingly

Reply:
This decorator can now be removed. LGTM otherwise

Reply:
That one slipped through, thanks!